### PR TITLE
Display draft taxons on topics pages

### PR DESCRIPTION
**Pull request by leenagupte**
Trello card: https://trello.com/c/NLT7h82S

## Motivation

Currently, only two taxons have been published, Education, and Childcare and parenting.
To be able to see different types of content tagged to a taxon, we need to be able to display more than these two taxons.

Rather than try to import all of the draft taxons from the draft-content-store and having to store them in a local database, we have decided to limit ourselves to two draft taxons, Transport and Money. We believe that these two taxons, along with the live taxons will give a large enough breadth of tagged document_types.

### CODE REVIEW

File: app/models/taxons.rb [hunk 1]
```diff
@@ -13,12 +13,23 @@ def live_level_one_taxons
   end
 
   def draft_level_one_taxons
-    draft_taxons.map do |taxon|
-      return taxon if taxon["links"]["parent_taxon"].blank?
-    end
+    draft_taxons.keep_if { |taxon| level_one_taxon?(taxon) }
```
Comment (fmlharrison) [inline app/models/taxons.rb]: Nice! 💪

File: app/models/taxon.rb [hunk 2]
```diff
@@ -26,7 +26,30 @@ def tagged_content_count(content_id)
   private
 
   def content_item
-    @content_item ||= Services.content_store.content_item(base_path)
+    @content_item ||= begin
+                        Services.content_store.content_item(base_path)
+                      rescue GdsApi::ContentStore::ItemNotFound
+                        draft_taxon
+                      end
```
Comment (fmlharrison) [inline app/models/taxon.rb:35]: This indentation is abit funky, no?

**Review by fmlharrison — COMMENTED** [review]
Looks good to me. Just one comment.

File: app/models/taxon.rb (hunk 2 above)
Comment (vanitabarrett) [reply app/models/taxon.rb:35]: I think it's okay, it just looks a bit strange because it's being assigned to the content_item variable?

**Review by vanitabarrett — APPROVED** [review]
Looks good to me! 

Just making note that this PR contains an issue where we are only returning 1000 content items but the count of content-items includes all. This has been written up in the following Trello card: https://trello.com/c/X8B0CKS8/67-return-all-tagged-content-from-rummager

File: app/models/taxon.rb (hunk 2 above)
Comment (fmlharrison) [reply app/models/taxon.rb:35]: Ah ok cool. Is this a Ruby formatting convention?

Comment (leenagupte) [reply app/models/taxon.rb:35]: I did it like that to make it clear that the outcome of the whole block is being assigned to the content_item variable. I don't think that it's a regular Ruby formatting convention, but it didn't fail the linter.

**Comment by leenagupte** [comment]
@fmlharrison @vanitabarrett One thing I've noticed after reviewing PR #17 is that it's not safe to assume that the draft taxon JSON files will be the only contents in `lib/data`. To prevent any future tests breaking I'll move the draft taxon files to sub-folder that is specifically for the JSON files.

**Review by fmlharrison — APPROVED** [review]
LGTM

**Pull request merged**